client/components/domains/domain-suggestion-flag/index.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+var React = require( 'react' );
+
+/**
+ * Internal dependencies
+ */
+var Flag = require( 'components/flag' );
+
+const DomainSuggestionFlag = React.createClass( {
+	propTypes: {
+		domain: React.PropTypes.string
+	},
+
+	render() {
+		var newTLDs = ['.live'];
+		var i;
+
+		if ( this.props.domain ) {
+			for ( i = 0; i < newTLDs.length; i ++ ) {
+				if ( this.props.domain.endsWith( newTLDs[i] ) ) {
+					return (
+						<Flag
+							type="is-success">
+							{ this.translate( 'New' ) }
```

Comment:
I think this string needs additional context to make it easier to properly translate it, like it's done, for example, [here](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/plugins/plugins-browser/index.jsx#L189).